### PR TITLE
Cm docstrings

### PR DESCRIPTION
**Pull request by david-deboer**
This updates all docstrings in files beginning with 'cm_' to the numpy standard.  It normalizes a few parameter names for consistency.  It adds one single script to update apriori

### CODE REVIEW

File: hera_mc/cm_utils.py [hunk 1]
```diff
@@ -199,8 +376,8 @@ def get_astropytime(_date, _time=0):
 
     Returns:  either astropy Time or None
 
-    Parameters:
-    -----------
+    Parameters
+    ----------
     _date:  date in various formats:
```
Comment (bhazelton) [inline hera_mc/cm_utils.py]: why are your parameters named with an initial underscore? Is it supposed to convey something meaningful to me?

File: hera_mc/cm_utils.py [hunk 2]
```diff
@@ -217,6 +394,10 @@ def get_astropytime(_date, _time=0):
     _time:  only used if _date is 'YYYY/M/D'/'YYYY-M-D' string otherwise ignored
                 float, int:  hours in decimal time
                 string:  HH[:MM[:SS]] or hours in decimal time
+
+    Returns
+    -------
+    Time or None
```
Comment (bhazelton) [inline hera_mc/cm_utils.py]: an astropy Time object?

File: hera_mc/cm_utils.py [hunk 3]
```diff
@@ -347,6 +552,20 @@ def csv_table(headers, table):
 
 
 def query_default(a, args):
+    """
+    Allows for a parameter to be queried, and return defaults for those not provided.
+
+    Parameters
+    ----------
+    a : str
```
Comment (bhazelton) [inline hera_mc/cm_utils.py]: no single letter variable names.

File: scripts/update_apriori.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Script to update the apriori antenna status.
+"""
+
+from hera_mc import mc, cm_utils, cm_partconnect
+
+
+if __name__ == '__main__':
+    parser = mc.get_mc_argument_parser()
+    parser.add_argument('-p', '--hpn', help="HERA part number", default=None)
+    parser.add_argument('-s', '--status', help="New apriori status enum.", default=None)
```
Comment (bhazelton) [inline scripts/update_apriori.py]: This isn't very helpful. Can you list the options from the enum?

File: INSTALL.md [hunk 5]
```diff
@@ -29,7 +29,7 @@ Python Prerequisites
  sqlalchemy |  psycopg2 | alembic | dateutil | numpy
  -----------|-----------|---------|----------|-------
  astropy    | tabulate  | pandas  | psutil   | pyproj
- six        | pyuvdata  | h5py    |          |
+ six        | pyuvdata  | h5py    | cartopy  |
```
[the remark this answers is not in the document]
Comment (david-deboer) [reply INSTALL.md]: It now just refers to setup.py to avoid inconsistencies.

File: hera_mc/cm_utils.py (hunk 1 above)
Comment (david-deboer) [reply hera_mc/cm_utils.py]: I've changed to 'adate' and 'atime' ('date' and 'time' are terrible variable names...)

**Comment by david-deboer** [comment]
Somehow I hadn't see those other comments.  I've addressed them, with a few comments on several.

**Comment by bhazelton** [comment]
@david-deboer it doesn't look like you pushed any new commits.

**Comment by david-deboer** [comment]
I have to make nosetests work again -- I'm not catching the new error I raise and I can't figure out why.  I'll take those out.

**Comment by david-deboer** [comment]
I'll push, but it should fail tests.

**Pull request merged**